build/release/latest_release_branch.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+# Get list of release- branches
+RELEASE_BRANCHES="$(git branch -r | grep -v mergify | grep '/release-' | cut -d'/' -f2 | sort | uniq)"
+LATEST_RELEASE_BRANCH=""
+MAX_TRIES=5
+
+TRY=1
+until [ -n "${LATEST_RELEASE_BRANCH}" ] || [ ${TRY} -ge ${MAX_TRIES} ]; do
+    LATEST_RELEASE_BRANCH="$(echo "${RELEASE_BRANCHES}" | sort --version-sort | tail -n 1 | cut -d'/' -f2)"
+    BRANCH_VERSION="v$(echo "${LATEST_RELEASE_BRANCH}" | cut -d'-' -f2-)"
+
+    # Get tags for the release version and filter out any alpha or beta versions
+    set +o pipefail
+    set +e
+    TAGS="$(git tag | grep -F "${BRANCH_VERSION}.0" | grep -Ev "alpha|beta")"
+    set -o pipefail
+    set -e
+
+    # If the list is empty remove last line from $RELEASE_BRANCHES and
+    # clear $LATEST_RELEASE_BRANCH var
+    if [ -z "${TAGS}" ]; then
+        LATEST_RELEASE_BRANCH=""
+        RELEASE_BRANCHES="$(echo "${RELEASE_BRANCHES}" | sed '$d')"
+    fi
+    (( TRY++ ))
+done
+
+if [ -z "${LATEST_RELEASE_BRANCH}" ]; then
+    echo "Failed to find latest released release branch!"
+    exit 1
```

Comment:
If we don't find the release branch, this will fail the build, right?

Reply:
Yes, it will fail if it doesn't find a release branch. One thing to consider would be to change the `MAX_RETRY` to just 2 so it will only check two branches (e.g., if the 1.11 release is in alpha it would check that and then "fallback" on the second try to the release-1.10 branch.

What do you think?

Reply:
Agreed, no need to check more than the most recent 2 branches as long as the sort is correct.